### ISSUE TITLE
performance of the trained model on the test set

### ISSUE THREAD

**Issue by jack-zzzzz** [ChenHongruixuan/BRIGHT#13]
Thank you very much for your help in training changemamba on the BRIGHT dataset.
 But why is the performance of the trained model on the test set very poor compared to the metrics reported in the paper?However the performance on the validation set is very good.Is this related to the Ukraine, Myanmar, and Mexico part of the data?

**Comment by ChenHongruixuan** [comment]
Hi @jack-zzzzz ,

Thank you so much for your question.

> But why is the performance of the trained model on the test set very poor compared to the metrics reported in the paper?However the performance on the validation set is very good.

May I ask what do you mean about this? Please provide more details, like your problem setup (machine learning split or cross-event transfer split). 

> Is this related to the Ukraine, Myanmar, and Mexico part of the data?

Have you preprocessed the optical data over these three areas sucessfully? 

Best,
